src/styles/fonts.js

```diff
@@ -0,0 +1,50 @@
+import { css } from 'styled-components';
```

Comment:
You can import Google fonts inside the `<head>` tag or a CSS file. For the source URL you can go to https://fonts.google.com/ select the fonts and click the banner `2 families selected`. You can even customize your selection and load only certain dunt weights